Packages/com.llealloo.audiolink/Runtime/Scripts/ytdlpPlayer.cs

```diff
@@ -155,36 +172,43 @@ public static bool IsytdlpAvailable()
         public static void Locateytdlp()
         {
             _ytdlpFound = false;
+
+            // check for a custom install location
+            string customPath = EditorPrefs.GetString(userDefinedYTDLPathKey, string.Empty);
+            if (!string.IsNullOrEmpty(customPath))
+            {
+                if (File.Exists(customPath))
+                {
+                    Debug.Log($"[AudioLink:ytdlp] Custom YTDL location found: {customPath}");
+                    _ytdlpPath = customPath;
+                    _ytdlpFound = true;
+                }
+
+                Debug.LogWarning($"[AudioLink:ytdlp] Custom YTDL location detected but does not exist: {customPath}");
```

Comment:
This doesn't seem right? There should be no warning printed if we found the executable in the branch above, right?

Reply:
Return statement is missing. Will fix.